DESCRIPTION

```diff
@@ -28,6 +29,7 @@ Suggests:
     patrick,
     pillar,
     rmarkdown,
+    spelling,
```

Comment:
Is this needed here?
I believe `Config/Needs/dev` is more appropriate as it will only be used during development.

Reply:
indeed, that was an accidental commit